controlplane/src/core/repositories/SubgraphRepository.ts

```diff
@@ -95,6 +96,7 @@ export class SubgraphRepository {
           organizationId: this.organizationId,
           labels: uniqueLabels.map((ul) => joinLabel(ul)),
           readme: data.readme,
+          asMonograph: data.asMonograph,
```

Comment:
This is another abstraction I find odd. A subgraph belongs to a federated graph but does not store metadata about the graph it belongs to. For me it feels like, that this information should not be needed, and the right subgraph is joined when selecting a target over the type `monograph`